clients/cli/src/create_escrow_account.rs

```diff
@@ -0,0 +1,153 @@
+    solana_pubkey::Pubkey,
+    solana_signature::Signature,
+    solana_signer::Signer,
+    solana_system_interface::instruction::create_account,
+    solana_transaction::Transaction,
+    spl_token_wrap::{get_wrapped_mint_address, get_wrapped_mint_authority},
+    std::fmt::{Display, Formatter},
+};
+
+#[derive(Clone, Debug, Args)]
+#[clap(about = "Creates an escrow token account for holding unwrapped tokens")]
+pub struct CreateEscrowAccountArgs {
+    /// The address of the mint for the unwrapped tokens the escrow will hold
+    #[clap(value_parser = parse_pubkey)]
+    pub unwrapped_mint: Pubkey,
+
+    /// The address of the token program for the *wrapped* mint
+    #[clap(value_parser = parse_token_program)]
+    pub wrapped_token_program: Pubkey,
+}
+
+#[serde_as]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CreateEscrowAccountOutput {
+    #[serde_as(as = "DisplayFromStr")]
+    pub escrow_account_address: Pubkey,
+
+    #[serde_as(as = "DisplayFromStr")]
+    pub escrow_account_owner: Pubkey, // This is the wrapped_mint_authority PDA
+
+    #[serde_as(as = "DisplayFromStr")]
+    pub escrow_token_program_id: Pubkey,
+
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    pub signature: Option<Signature>,
+}
+
+impl Display for CreateEscrowAccountOutput {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        writeln_name_value(
+            f,
+            "Escrow Account Address:",
+            &self.escrow_account_address.to_string(),
+        )?;
+        writeln_name_value(
+            f,
+            "Escrow Account Owner (PDA):",
+            &self.escrow_account_owner.to_string(),
+        )?;
+        writeln_name_value(
+            f,
+            "Escrow Token Program ID:",
+            &self.escrow_token_program_id.to_string(),
+        )?;
+        if let Some(signature) = self.signature {
+            writeln_name_value(f, "Signature:", &signature.to_string())?;
+        }
+        Ok(())
+    }
+}
+
+impl QuietDisplay for CreateEscrowAccountOutput {
+    fn write_str(&self, _: &mut dyn std::fmt::Write) -> std::fmt::Result {
+        Ok(())
+    }
+}
+impl VerboseDisplay for CreateEscrowAccountOutput {}
+
+pub async fn command_create_escrow_account(
+    config: &Config,
+    args: CreateEscrowAccountArgs,
+) -> CommandResult {
+    let payer = config.fee_payer()?;
+    let rpc_client = config.rpc_client.clone();
+
+    // --- Determine Unwrapped Token Program ---
+    let unwrapped_token_program_id =
+        get_account_owner(&args.unwrapped_mint, rpc_client.clone()).await?;
+
+    // --- Derive PDAs ---
+    let wrapped_mint_address =
+        get_wrapped_mint_address(&args.unwrapped_mint, &args.wrapped_token_program);
+    let wrapped_mint_authority = get_wrapped_mint_authority(&wrapped_mint_address);
+
+    println_display(
+        config,
+        format!(
+            "Creating escrow account under program {} for unwrapped mint {} owned by PDA {}",
```

Comment:
Up to you, for more clarity:

```suggestion
"Creating escrow account under program {} for unwrapped mint {} owned by wrapped mint authority {}",
```